app/tut/ga/cluiche.md

```diff
@@ -0,0 +1,701 @@
+- airde_slacáin`{.setanta}.
+
+![y_slacáin](assets/y_slacan.png)
+
+Anois seo é ár ríomhchlár [[reatha|current]].
+
+```{.setanta .numberLines}
+airde_slacáin := 20
+lthd_slacáin := fad_x@stáitse // 5
+
+x_slacáin := 0
+y_slacáin := fad_y@stáitse - airde_slacáin
+
+gníomh tarraing_stáitse() {
+    >-- Cuirfimid ár cód tarraingthe anseo.
+}
+
+>-- Lúb go deo.
+nuair-a fíor {
+    >-- Glaoigh ar an gníomh tarraing_stáitse.
+    tarraing_stáitse()
+
+    >-- Codail ar feadh cúple milleasoicind.
+    codladh(10)
+}
+```
+
+## Tarraing an slacán
+
+Anois tá a fhios againn cá bhfuil ár slacán (`x_slacáin`, `y_slacáin`) agus
+[[cé chomh mór atá sé|how big it is]] (`airde_slacáin`, `lthd_slacáin`), is féidir linn é a
+tharraingt.
+
+Sula dtarraingeoimid ár slacán, ba chóir dúinn an stáitse a ghlanadh chun aon
+[[sean-rudaí|old things]] a [[scriosadh|erase]]. Bainimid úsáid as an gníomh
+`glan@stáitse`{.setanta} chun é sin a dhéanamh.
+
+Tar éis é sin, is féidir linn [[dath an pinn|colour of the pen]] a athrú. Ba mhaith liom slacán
+dearg a tharraingt, mar sin athróidh mé an dath go dearg le `dath@stáitse("dearg")`{.setanta}.
+
+Anois táimid in ann an gníomh `dron_lán@stáitse`{.setanta} a úsáid chun
+[[**dron**uilleog **lán**|full rectangle]] a tharraingt le haghaidh an slacán. Glacann sé le 4
+argóint: an x-comhordanáid, an y-comhordanáid, an leithead agus an airde:
+
+Anois seo é ár gníomh `tarraing_stáitse`:
+
+```{.setanta .numberLines}
+gníomh tarraing_stáitse() {
+    >-- Glan an stáitse
+    glan@stáitse()
+
+    >-- Úsáid peann dearg
+    dath@stáitse("dearg")
+
+    >-- Tarraing an slacán
+    dron_lán@stáitse(x_slacáin, y_slacáin, lthd_slacáin, airde_slacáin)
+}
+```
+
+Anois bainimis triail as ár ríomhchlár:
+
+{{{s
+airde_slacáin := 20
+lthd_slacáin := fad_x@stáitse // 5
+
+x_slacáin := 0
+y_slacáin := fad_y@stáitse - airde_slacáin
+
+gníomh tarraing_stáitse() {
+    >-- Glan an stáitse
+    glan@stáitse()
+
+    >-- Úsáid peann dearg
+    dath@stáitse("dearg")
+
+    >-- Tarraing an slacán
+    dron_lán@stáitse(x_slacáin, y_slacáin, lthd_slacáin, airde_slacáin)
+}
+
+>-- Lúb go deo.
+nuair-a fíor {
+    >-- Glaoigh ar an gníomh tarraing_stáitse.
+    tarraing_stáitse()
+
+    >-- Codail ar feadh cúple milleasoicind.
+    codladh(10)
+}
+}}}
+
+**Ná dean dearmad an [[cnaipe|button]] <iron-icon class="play" icon="av:stop"></iron-icon> a bhrúigh
+chun an ríomhchlár a stopadh**
+
+Bá chóir duit slacán beag dearg ag bun an stáitse a fheiceáil.
+
+## Bog an slacán
+
+Anois is féidir linn an slacán a tharraingt ach conas a bhogaimid é? Is féidir linn an gníomh
+[[`méarchlár`|keyboard]] a úsáid chun gach [[eochairbhrú|keypress]] a fháil ón
+[[úsáideoir|user]].
+
+Ar dtús, cruthaímis gníomh nua agus tugaimis `smacht_eochrach` air. Glaofar ar an gníomh seo nuair a
+brúnn an úsáideoir [[eochair|key]] ar an méarchlár. Bá chóir dó glacadh le argóint amháin: an
```

Comment:
le hargóint amháin https://www.teanglann.ie/ga/gram/_p_n?p=le_prep&n=arg%c3%b3int_fem2